README.md

```diff
@@ -1,2 +1,25 @@
 # import_service_terraform
 Terraform for import service
+
+Your .tfvars file should contain the following:
+
+**variable**|**description**
+-----|-----
+`env`                                             | used to generate a lot of defaults
+`import_service_google_project_folder_id`    | the folder ID to put the project in, e.g. `"123456789012"`
+`billing_account_id`                            | the google billing account to link the project to, e.g. `"XXXXXX-XXXXXX-XXXXXX"`
+`terraform_google_project`                      | the google project owning the service account that Terraform runs as. can be empty if running locally
+`audience_domain`                                | used as a semi-secret by pub/sub. must match import service environment variable
+`rawls_sa_email`                                | email address of the rawls service account in this env
+`sam_sa_email`                                  | email address of the sam service account in this env
+`terra_google_project`                          | google project that terra monolithic services run in (i.e. where rawls and sam SAs live)
+
+
+Your .tfvars MAY contain the following, if you want to override the defaults during local development:
+
+
+**variable**|**default**|**description**
+-----|-----|------
+`import_service_google_project` | `{var.env}` | The Google project to create, `terra-importservice-{import_service_google_project}`
+`bucket_suffix` | `{var.env}` | Creates the import service batchUpsert bucket, `importservice-batchupsert-{bucket_suffix}`
```

Comment:
defaults for `import_service_google_project` and `bucket_suffix` seem wrong

Reply:
the default i listed here is the default value for the variable, not the final value of the generated string. it's easier to see for the bucket suffix (if you do nothing, the bucket will be called `importservice-batchupsert-yourenv`) but i see now that this is all a bit confusing. i'll update some things, thank you

Reply:
wow holy shit this is garbage

Reply:
okay, try again? i think bucket_suffix is fine, tell me why i'm wrong?